### PR TITLE
Rap8 - Added line before threatcrowd and anchors to jump around the page

### PR DESCRIPTION
**Pull request by bauerm**

### CODE REVIEW

**Comment by sbgoodm** [comment]
@bauerm want to fix the bug with install.sh in this PR too?

**Comment by bauerm** [comment]
Sure, done.

File: templates/pivoteer/RecentRecords.html [hunk 1]
```diff
@@ -130,9 +133,22 @@
     <h4>No Records Found</h4>
 {% endif %}
 
+<style>
+.linebreak {
```
Comment (sbgoodm) [inline templates/pivoteer/RecentRecords.html]: Move this linebreak CSS class into static/css/RAPID.css. You'll still be able to reference `<hr class="linebreak">`, but we'll consolidate the CSS definitions in the same place.

**Comment by bauerm** [comment]
Excellent point, I've made that change.

**Pull request merged**